test/generate_markdown.py

```diff
@@ -0,0 +1,159 @@
+import os
+import re
+import pandas as pd
+from typing import NewType
+import sys
+import argparse
+
+try:
+    from ..main import run, ExecutionMode, do_try
+    from .utils import collect_test_files
+except ImportError:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+    from main import run, ExecutionMode, do_try
+    from test.utils import collect_test_files
+
+
+MarkdownTableStr = NewType('MarkdownTableStr', str)
+
+def handle_markdown_table(markdown_table: MarkdownTableStr) -> pd.DataFrame:
+    """
+    Handling in case the already existing file is in non-html markdown format
+    """
+    lines = [line.strip() for line in markdown_table.split("\n")]
```

Comment:
You can use `markdown.markdown()` (`pip install markdown`) to convert the markdown table to html.

Reply:
We're now using html and converting it to markdown only in the end, so this is not required anymore. 
[update markdown generate script](https://github.com/buildingSMART/ifc-gherkin-rules/pull/177/commits/1ea22a7c634b248eb857d40f1831d26b57aa9c74)